src/index.ts

```diff
@@ -605,6 +607,89 @@ export async function precomputedInit(
   return EppoPrecomputedJSClient.instance;
 }
 
+/**
+ * Initializes the Eppo precomputed client with configuration parameters.
+ *
+ * The purpose is for use-cases where the precomputed assignments are available from an external process
+ * that can bootstrap the SDK.
+ *
+ * This method should be called once on application startup.
+ *
+ * @param config - client configuration
+ * @returns a singleton precomputed client instance
+ * @public
+ */
+export interface IPrecomputedClientConfigSync {
+  precomputedConfigurationWire: string;
+  assignmentLogger?: IAssignmentLogger;
+  throwOnFailedInitialization?: boolean;
+}
+
+/**
+ * Initializes the Eppo precomputed client with configuration parameters.
+ *
+ * The purpose is for use-cases where the precomputed assignments are available from an external process
+ * that can bootstrap the SDK.
+ *
+ * This method should be called once on application startup.
+ *
+ * @param config - precomputed client configuration
+ * @returns a singleton precomputed client instance
+ * @public
+ */
+export function offlinePrecomputedInit(
+  config: IPrecomputedClientConfigSync,
+): EppoPrecomputedClient {
+  const throwOnFailedInitialization = config.throwOnFailedInitialization ?? true;
+
+  const configurationWire: IConfigurationWire = JSON.parse(config.precomputedConfigurationWire);
+  if (!configurationWire.precomputed) {
+    applicationLogger.error('Invalid precomputed configuration wire');
+    return EppoPrecomputedJSClient.instance;
+  }
+  const {
+    subjectKey,
+    subjectAttributes: contextAttributes,
+    response,
+  } = configurationWire.precomputed;
+  const parsedResponse: IPrecomputedConfigurationResponse = JSON.parse(response);
+
+  try {
+    const memoryOnlyPrecomputedStore = precomputedFlagsStorageFactory();
+    memoryOnlyPrecomputedStore
+      .setEntries(parsedResponse.flags)
+      .catch((err) =>
+        applicationLogger.warn('Error setting precomputed assignments for memory-only store', err),
+      );
+
+    const subjectAttributes: Attributes = convertContextAttributesToSubjectAttributes(
+      contextAttributes ?? { numericAttributes: {}, categoricalAttributes: {} },
+    );
+
+    EppoPrecomputedJSClient.instance.setSubjectSaltAndPrecomputedFlagStore(
```

Comment:
Based on these PR comments 

https://github.com/Eppo-exp/js-sdk-common/pull/164#discussion_r1905896857
https://github.com/Eppo-exp/js-sdk-common/pull/164#discussion_r1905899272

I'll see if I can make this method private/not use it here and have these set through the constructor